google/cloud/forseti/services/inventory/crawler.py

```diff
@@ -420,7 +422,7 @@ def run_crawler(storage,
         LOGGER.info('SQLite used, disabling parallel threads.')
         parallel = False
 
-    client = _api_client_factory(storage, config, parallel)
+    client = _api_client_factory(storage, config, parallel, tracer)
```

Comment:
No need to pass `tracer` explicitly here, since this method call is in the same context as the calling method. The tracer will be inferred from context and set to the `tracer` argument of the method automatically.

Reply:
Done.